packages/utils/schema/user-profile.graphql

```diff
@@ -31,6 +34,7 @@ type ImageMetadata {
   # Dimensions for raster images
   width: Int @int(min: 0)
   height: Int @int(min: 0)
+
   # For vector images
-  aspectRatio: Float @float(min: 0, max: 1)
+  aspectRatio: Float @float(min: 0, max: 2)
```

Comment:
It seems like the aspect ratio can be of arbitrary maximum size. Discord banners are 800:320 which is 2.5.